scripts/packaging/createupmpackages.ps1

```diff
@@ -92,7 +99,7 @@ if ($nodejsInstalled -eq $false)
     Write-Output "Extracting $archiveFile"
     Expand-Archive -Path $archiveFile -DestinationPath ".\" -Force
 
-    $npmPath = "$scriptPath\$archiveName\$npmPath"
+    $npmCommand = "$scriptPath\$archiveName\$npmPath\$npmCommand"

```

Comment:
Actually... we should delete these lines that acquire nodejs in this location.

We should not be reaching out to external web dependencies in a build pipeline outside of the specific packages provided by the containers on that CI system.

Please delete lines 91 to 101

Reply:
the build pipeline does not use these, they are for the local runs. that said, i am happy to remove

Reply:
done